about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- bootstrap link -->
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+  <!-- font awesome -->
+
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p
+    " crossorigin="anonymous" />
+  <title>DDT </title>
+  <link rel="icon" href="./asset/image/logo_mobile.png">
+  <link rel="stylesheet" href="./asset/scss-about/about.css">
+  <!-- <link rel="stylesheet" href="./css/style.css"> -->
+</head>
+
+<body>
+  <header class="flex-column-center">
+
+    <nav>
+      <!-- header and nav for mobile -->
+      <div class="nav-bg"></div>
+      <div class="mobile_version hide_me">
+        <div class="openPop" id="openPop"><i class="fas fa-bars fa-2x"></i></div>
+        <ul class="hidepop list nav-links" id="list">
+          <li class="closep"><a href="./index.html">Home</a></li>
+          <li class="closePop"><a href="#logo">Program</a></li>
+          <li class="closePop"><a href="#">Join</a></li>
+          <li class="closePop"><a href="#sponsor">Sponsor</a></li>
+          <li class="closeMenu" id="closeMenu"><a href="#topic"><i class="fas fa-times fa-2x"></i></a></li>
+        </ul>
+      </div>
+      <!-- header and nav for desktop version  -->
+
+      <div class="desktop-version top-nav">
+        <div class="destop-version icon">
+          <ul class="icon_list">
+            <li><i class="fab fa-facebook-f"></i></li>
+            <li><i class="fab fa-twitter"></i></li>
+            <li>
+              <p>English</p>
+            </li>
+            <li>
+              <p>My Page</p>
+            </li>
+            <li>
+              <p>Logout</p>
+            </li>
+          </ul>
+        </div>
+        <div class="navigation">
+          <div class="main-logo">
+            <img src="./asset/image/ddt.png" alt="" />
```

Comment:
- [ ] Kindly wrap the logo image inside a `<a>` tag in order to be a link that takes the user to the home page. (Consider fixing this issue on both pages).